readme.txt

```diff
@@ -5,7 +5,7 @@ Tags: donation, donations, donation plugin, wordpress donation plugin, givewp, g
 Requires at least: 4.8
 Tested up to: 5.5
 Requires PHP: 5.6
-Stable tag: 2.7.5
+Stable tag: 2.8.0
```

Comment:
I don't think this is necessary.

Reply:
fixed in https://github.com/impress-org/givewp/compare/dec997d3b90a...613679045944